testsuite/unittests/test_gif_cond_exp.sli

```diff
@@ -0,0 +1,112 @@
+/*
+ *  test_gif_cond_exp.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+ /* BeginDocumentation
+Name: testsuite::test_gif_cond_exp - sli script for test of gif_cond_exp with external DC current and spike generators
+
+Synopsis: (test_gif_cond_exp) run
+
+
+Description:
+
+ This testscript checks the correctness of gif_psc_exp neuron model. It injects external DC current and also three spikes to two neurons and measures their spike times. The spike times should match the expected values.
+
+Author:  March 2016, Setareh
+SeeAlso: gif_cond_exp, testsuite::test_gif_cond_exp_multisynapse
+*/
+
+% This test should only run if we have GSL
+statusdict/have_gsl :: not {statusdict/exitcodes/success :: quit_i} if
+
+(unittest) run
+/unittest using
+
+0.1 /h Set
+
+ResetKernel
+
+0 << 
+      /local_num_threads 1 
+      /resolution h
+      /grng_seed 0
+      /rng_seeds [0]
+    >> SetStatus
+
+% neuron1 should fire spikes at 42.6, 84.9 and 122.7 ms
+426  /t1_neuron1  Set 
+849  /t2_neuron1  Set
+1227 /t3_neuron1  Set
```

Comment:
I saw now that you comment on that a while ago here in the PR. But I think it is important that you put the information about the reference solution into the test code here, preferably with a link to the repo holding the Python reference code by the original authors.